src/services/API/Item.js

```diff
@@ -45,4 +49,15 @@ export default {
       plioId: plioDBId,
     });
   },
+
+  /**
+   * Delete a list of items
+   * @param {Object} payload - params required for deleting the required items
+   */
+  bulkDelete(payload) {
+    // slice is done to avoid // in the endpoint
```

Comment:
Why not keep the `bulkDeleteEndpoint` as `bulk_delete/` instead of `/bulk_delete/`?
Bulk delete endpoint will only be called for an asset right? Like with item. So it will always be appended to an existing asset's endpoint.

I'm saying this because this is what we did with list uuid endpoint.

Reply:
see where this can be applied